### PR TITLE
dockerfile/termui: add Dockerfile for termui

### PR DESCRIPTION
**Pull request by Meria-finance**
## Reasoning behind the pull request
- no docker image for termui is available
  
## Proposed changes
- adding Dockerfile for termui

## Testing procedure
- we've locally tested the docker image

### CODE REVIEW

**Review by miiu96 — APPROVED** [review]

File: docker/termui/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.17.6 as builder
+RUN apt-get update && apt-get install -y
+WORKDIR /go/mx-chain-go
+COPY . .
+WORKDIR /go/mx-chain-go/cmd/termui
+RUN go build -i -v
+RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/mx-chain-go/go.mod | grep arwen-wasm-vm | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so || :
```
Comment (iulianpascalau) [inline docker/termui/Dockerfile:7]: I do not think we need the libwasmer_linux_amd64.so file management. So L7 & L11 can be removed

Comment (iulianpascalau) [reply docker/termui/Dockerfile:7]: LE: no, it does not work without those lines, will refactor them when we will do the merge from `master` -> `rc/v1.4.0`

**Review by iulianpascalau — CHANGES_REQUESTED** [review]
commit is not verified. We will do the changes by our own. For future PRs please ensure that your commits are verified. Will close this PR.

**Comment by miiu96** [comment]
Hello 
We have created a new PR (https://github.com/multiversx/mx-chain-go/pull/4940)  for your request (we cannot merge this one because it has an unsigned commit). 
Thank you for this PR and for your contribution. 
This PR will be closed.

**Pull request closed without merging**